app/common/lib/ledgerUtil.js

```diff
@@ -303,7 +471,18 @@ const getMediaProvider = (url) => {
 
   // Youtube
   if (url.startsWith('https://www.youtube.com/api/stats/watchtime?')) {
-    provider = ledgerMediaProviders.YOUTUBE
+    return ledgerMediaProviders.YOUTUBE
+  }
+
+  // Twitch
+  if (
+    (
+      (firstPartyUrl && firstPartyUrl.startsWith('https://www.twitch.tv')) ||
```

Comment:
food for thought- with more integrations being added, it might make sense to move the URLs into `ledgerMediaProviders.js`.  That module could then export:
- type (ex: YOUTUBE, TWITCH)
- providerUrls (indexed by type)

Reply:
this url's are used only here, so I wouldn't add it into the providers